data.py

```diff
@@ -0,0 +1,83 @@
+import csv
+import json
+import operator
+
+import nlu
+import questions
+
+
+class Similarity(object):
+    def __init__(self, intent: str, score: float):
+        self.intent = intent
+        self.score = score
+
+    def __repr__(self):
+        return '<Similarity {intent} {score}>'.format(
+            intent=self.intent, score=self.score)
+
+
+class SimilarityResult(object):
```

Comment:
It's more like a report; later we base on this report to pick a command, what do you think?